gradle/dependencies.gradle

```diff
@@ -39,7 +39,7 @@ versions += [
   rocksDB: "4.1.0",
   scalaTest: "2.2.6",
   scalaParserCombinators: "1.0.4",
-  slf4j: "1.7.15",
+  slf4j: "1.7.16",
```

Comment:
It seems like 1.7.18 has hit Maven since then:

http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.slf4j%22%20AND%20a%3A%22slf4j-api%22

Not sure what's going on with slf4j.